src/lib.rs

```diff
@@ -394,6 +394,30 @@ where
     runner.run(f);
 }
 
+/// Sets the `tag` field of the current task.
+pub fn set_tag(tag: u64) {
+    use runtime::execution::ExecutionState;
```

Comment:
Could we avoid introducing all these `use` statements that take up screen real estate? Either move these `use` up to the top, or just write `runtime::execution::ExecutionState::set_tag`